PubNub/Data/Managers/PNPublishSequence.m

```diff
@@ -369,30 +376,6 @@ - (void)subscribeOnNotifications {
 #endif // TARGET_OS_OSX
 }
 
-- (void)unsubscribeFromNotifications {
```

Comment:
I don't know, comment refer to removed `-unsubscribeFromNotifications` so I respond on it.
But there is no `UIApplicationDidResignActive` in SDK :)
Also, nothing changed compared to master in `-subscribeOnNotifications` and both: `UIApplicationWillResignActiveNotification` and `UIApplicationDidEnterBackgroundNotification` there.